frontend/src/views/CellGuide/components/CellGuideCard/style.ts

```diff
@@ -204,7 +220,7 @@ export const MobileTooltipWrapper = styled.div`
   display: flex;
   flex-direction: column;
   height: calc(100vh - ${HEADER_HEIGHT_PX}px);
-  overscroll-behavior: contain;
+  overscroll-behavior: none;
```

Comment:
the same as contain except also removes native browser scroll-chaining (e.g. the rubber-band effect when scrolling to the end)